mappings/net/minecraft/data/server/RecipesProvider.mapping

```diff
@@ -276,6 +276,8 @@ CLASS net/minecraft/unmapped/C_hwlnmobw net/minecraft/data/server/RecipesProvide
 		ARG 0 cache
 		ARG 1 json
 		ARG 2 path
+	METHOD m_uvcwjlge (Ljava/util/Set;Lnet/minecraft/unmapped/C_sdjwtmhs;Ljava/nio/file/Path;Lnet/minecraft/unmapped/C_qoyuyccf;)V
+		ARG 3 recipe
```

Comment:
Are there no other arguments to this method?

Reply:
Nope, it's a lambda passed to `RecipesProvider#generate`